src/runtest.py

```diff
@@ -0,0 +1,189 @@
+                              cpus=0.01, mem=32,
+                              ports=[0],
+                              constraints=constraints)
+        if self.options.total_client_count > max_threads_per_client:
+            client_count = math.ceil(self.options.total_client_count / 10)
+            l.info("Number of Cassandra-Stress Clients to launch = %s" % (client_count))
+            self.scale_and_verify_app(self.stress_client, client_count)
+
+        ipm = self.get_app_ipport_map(self.stress_client)
+        # TODO: THIS WILL BE AN ARRAY OF CLIENT IPs
+        self.client_ip = ipm.values()[0][1]
+        self.client_rep_taskport = str(ipm.values()[0][0])
+
+        l.info("Cassandra Stress Client running at [%s]", self.client_ip)
+        l.info("Cassandra Stress Client - (HDaemon) REP Server running at [%s:%s]", self.client_ip, self.client_rep_taskport)
+
+        # Initialize CassandraAnalyser (to control & collect stats from test instances)
+
+        # self.ha_stress_client = HAnalyser(self.client_ip, self.client_rep_taskport, ipm.keys()[0])
+
+    def delete_all_launched_apps(self):
+        l.info("Deleting Stress Clients")
+        self.delete_app(self.stress_client)
+
+
+class RunTest(object):
+    def __init__(self, argv):
+        usage = ('python %prog --test_duration=<time to run test> --total_ops_count=<Total Operations>'
+                 '--total_client_count=<Total clients to launch> --cluster_ips=<cassandra node list ips>'
+                 '--consistency_level=<cassandra consistency level> --profile=<yaml profile>'
+                 '--config_file=<path_to_config_file>')
+        parser = OptionParser(description='cassandra scale test master',
+                              version="0.1", usage=usage)
+        parser.add_option("--test_duration", dest='test_duration', type='float', default=10)
+        parser.add_option("--total_ops_count", dest='total_ops_count', type='int', default=1000000)
+        parser.add_option("--total_client_count", dest='total_client_count', type='int', default=20)
+        parser.add_option("--cluster_ips", dest='cluster_ips', type='string', default='127.0.0.1')
+        parser.add_option("--consistency_level", dest='cl', type='string', default='LOCAL_ONE')
+        parser.add_option("--profile", dest='profile', type='string', default='hydra_profile.yaml')
+        parser.add_option("--config_file", dest='config_file', type='string', default='hydra.ini')
+
+        (options, args) = parser.parse_args()
+        # Check NO list of positional arguments leftover after parsing options
+        if ((len(args) != 0)):
+            parser.print_help()
+            sys.exit(1)
+
+        # Run Cassandra Test
+        r = RunTestCassandra(options, False)
+        r.start_appserver()
+        res = r.run_test()
+        r.delete_all_launched_apps()
+        # Cassandra-Stress Test Results
+        l.info("Cassandra Stress Results: \n%s" % pformat(res))
+        r.stop_appserver()
```

Comment:
Add __main__ section so that it can be run independently without running hyda form command line. 
You can look up in helloworld app for hint.